torch/_dynamo/symbolic_convert.py

```diff
@@ -2721,7 +2771,7 @@ def __init__(
                 if k in f_locals
             }
 
-            self._init_torch_function_mode_stack()
+            self._init_torch_function_mode_stack(torch_function_mode_stack)
```

Comment:
How do you deal with InliningInstructionTranslator? Does it have its own symbolic_torch_function_mode_stack?

Reply:
No this is a global stack that is shared. This makes sense, because there is only one stack, that is pushed/popped from even while inlining. This is hanlded by passing a reference to the symbolic stack to the InliningInstructionTranslator.